src/constants/url.js

```diff
@@ -0,0 +1,3 @@
+const url = "https://ecommerce-champions.onrender.com";
+
+export default url;
```

Comment:
@sostene12 Vite exposes env variables on the special `import.meta.env` object so we can put the api url into `.env.local` file. Kindly follow this [link](https://vitejs.dev/guide/env-and-mode.html) for Env Variables and Modes